### PR TITLE
Add a subcommand to compute stats on sequence file

### PR DESCRIPTION
**Pull request by natir**
Hi,

This pull request add a subcommand to compute statistics on sequence provides in stdin.

I write statistical function by my self, maybe use an external crate is better, what do you think on this?

Maybe add an option to add a csv output format could be a good idea.

Thanks for your work.

### CODE REVIEW

File: src/cli.yaml [hunk 1]
```diff
@@ -306,3 +306,17 @@ subcommands:
                   help: Add list of reads that were merged for each consensus read. Note that this can yield very long FASTQ name lines which cannot be handled by some tools.
 
 
+  - sequences-stats:
+      about: |
+        Tool to compute stats on sequence file (from STDIN).
```
Comment (johanneskoester) [inline src/cli.yaml:311]: Thanks a lot! I think the help text here should give some detail about what exactly is computed and printed in which format.

File: src/sequences_stats.rs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    let nb_bases = lengths.iter().sum::<usize>();
+
+    println!(
+        "min: {min}
+max: {max}
+average: {average}
+mediane: {mediane}
+number of reads: {nb_reads}
+number of bases: {nb_bases}
+n50: {n50}",
```
Comment (johanneskoester) [inline src/sequences_stats.rs:56]: Please mention that output is in YAML format in the help text.

Comment (natir) [reply src/sequences_stats.rs:58]: It's done

**Comment by natir** [comment]
up :)

File: src/cli.yaml [hunk 3]
```diff
@@ -322,11 +322,18 @@ subcommands:
 
   - sequences-stats:
```
Comment (johanneskoester) [inline src/cli.yaml:323]:
```suggestion
  - sequence-stats:
```

File: src/cli.yaml [hunk 4]
```diff
@@ -322,11 +322,18 @@ subcommands:
...
       about: |
-        Tool to compute stats on sequence file (from STDIN).
+        Tool to compute stats on sequence file (from STDIN), output is in YAML with field:
```
Comment (johanneskoester) [inline src/cli.yaml:325]:
```suggestion
        Tool to compute stats on sequence file (from STDIN), output is in YAML with fields:
```

File: src/cli.yaml [hunk 5]
```diff
@@ -322,11 +322,18 @@ subcommands:
...
+          - median: median length of sequence
+          - nb_reads: number of reads
+          - nb_bases: number of bases
+          - n50: N50 of sequences
 
         Example:
-          rbt sequence-stat < test.fasta
-          rbt sequence-stat -q < test.fastq
+          rbt sequences-stats < test.fasta
+          rbt sequences-stats -q < test.fastq
```
Comment (johanneskoester) [inline src/cli.yaml:336]:
```suggestion
          rbt sequence-stats < test.fasta
          rbt sequence-stats -q < test.fastq
```

**Comment by johanneskoester** [comment]
Thank you! And sorry for the delay.

**Comment by natir** [comment]
No problem, thank for you work !

**Pull request merged**